src/topology/sheaves/locally_surjective.lean

```diff
@@ -0,0 +1,116 @@
+/-
+Copyright (c) 2022 Sam van Gool and Jake Levinson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sam van Gool, Jake Levinson
+-/
+
+import topology.sheaves.presheaf
+import topology.sheaves.stalks
+
+/-!
+
+Locally surjective maps of presheaves.
+
+Let X be a topological space, ℱ and 𝒢 presheaves on X, T : ℱ ⟶ 𝒢 a map.
+
+In this file we formulate two notions for what it means for
+T to be locally surjective:
+
+  1. For each open set U, each section t ∈ 𝒢(U) is in the image of T
```

Comment:
```suggestion
  1. For each open set U, each section t : 𝒢(U) is in the image of T
```
We usually use `∈` only when the RHS is a substructure, and the LHS is an element of the parent structure that happens to fall in the substructure.